src/mars.c

```diff
@@ -1193,6 +1194,7 @@ Language changes listed by -transition=id:\n\
         VersionCondition::addPredefinedGlobalIdent("D_NoBoundsChecks");
 
     VersionCondition::addPredefinedGlobalIdent("D_HardFloat");
+    objc_tryMain_dObjc();
```

Comment:
Jacob did this after Walter asked for Objective-C related things to be put in separate functions. Personally, I find it more confusing than useful. Those functions could have more meaning full names I guess, but all those parts weren't originally meant to be standalone functions in the first place.

If it was only me, I'd just call `VersionCondition::addPredefinedGlobalIdent("D_ObjectiveC")` here and delete `objc_tryMain_dObjc` (it was like this in the original). But with the only advice from Walter being that things must be kept segregated in functions, I'm not sure what Jacob can do to make keep this patch acceptable. Perhaps a more verbose name such as `objc_addObjectiveCPredefinedGlobalIdent` would make the situation better.